tests/e2e/data-importers/pxc/11-assert.yaml

```diff
@@ -4,7 +4,7 @@ timeout: 300
 collectors:
   - command: kubectl get db/test-pxc-cluster -n ${NAMESPACE} -o yaml
   - command: kubectl get pxc/test-pxc-cluster -n ${NAMESPACE} -o yaml
-  - command: kubectl describe pods -n ${NAMESPACE}
```

Comment:
What's the reason for removing this? The `describe` contains lots of useful information for why the pods do not come up.

Reply:
the result of moving file 12 -> 11